src/modules/rlm_yubikey/rlm_yubikey.c

```diff
@@ -75,7 +75,7 @@ fr_dict_attr_autoload_t rlm_yubikey_dict_attr[] = {
 	{ .out = &attr_yubikey_public_id, .name = "Vendor-Specific.Yubico.Yubikey-Public-ID", .type = FR_TYPE_STRING, .dict = &dict_radius },
 	{ .out = &attr_yubikey_private_id, .name = "Vendor-Specific.Yubico.Yubikey-Private-ID", .type = FR_TYPE_OCTETS, .dict = &dict_radius },
 	{ .out = &attr_yubikey_counter, .name = "Vendor-Specific.Yubico.Yubikey-Counter", .type = FR_TYPE_UINT32, .dict = &dict_radius },
-	{ .out = &attr_yubikey_timestamp, .name = "Vendor-Specific.Yubico.Yubikey-Timestamp", .type = FR_TYPE_UINT32, .dict = &dict_radius },
+	{ .out = &attr_yubikey_timestamp, .name = "Vendor-Specific.Yubico.Yubikey-Timestamp", .type = FR_TYPE_TIME_DELTA, .dict = &dict_radius },
```

Comment:
https://developers.yubico.com/OTP/OTPs_Explained.html

says that the timestamp is a 24 bit incrementing field, so it's not an actual clock :(.  It should then be left as `uint32`.

Reply:
Alright, I did all changes. thanks @alandekok